examples/pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/ipex/main.py

```diff
@@ -106,6 +106,8 @@
                     help='run benchmark')
 parser.add_argument('--ipex', dest='ipex', action='store_true',
                     help='tuning or benchmark with Intel PyTorch Extension')
+parser.add_argument('--xpu', action='store_true',
+                    help='whether use xpu')
```

Comment:
Please add README how to use and how to install xpu.
Could you unity script(cpu, gpu, xpu) by args parser, the path show "cpu", is it limitation?

Reply:
This xpu args is for internal test. @violetch24 will take own of examples enabling.
The env building of XPU is not ready to share due to lots of local resources are used.